zeppelin-web/README.md

```diff
@@ -19,19 +19,27 @@ It is recommended to install node 6.0.0+ since Zeppelin uses 6.9.1+ (see [creati
 
 All build commands are described in [package.json](./package.json)
 
-```sh
 # install required depepdencies and bower packages (only once)
-$ npm install -g yarn
-$ yarn install
+```
+$ npm install -g yarn bower npm-run-all rimraf grunt webpack karma karma-phantomjs-launcher
```

Comment:
@DmytroShkvyra We don't need to install these packages as global.  All packages installed locally (`node_modules`) can be used in `scripts` tag in `package.json` as command like `grunt`, `webpack`